.github/workflows/generate-check-regression-reports.yml

```diff
@@ -9,29 +9,29 @@ jobs:
     runs-on: ubuntu-latest
     container: checkstyle/jdk-11-groovy-git-mvn:11.0.13__3.0.9__2.25.1__3.6.3
     env:
-      MAVEN_OPTS: -Xmx4096m
-      JAVA_OPTS: -Xmx4096m
+      MAVEN_OPTS: -Xmx5g
+      JAVA_OPTS: -Xmx5g
```

Comment:
@nrmancuso Just want to confirm will this increase the memory size and help to generate report which fail in checkstyle pr because memory error

Reply:
Are you asking a question or making a statement?

Reply:
Sorry for unclear :)

I am asking

Reply:
Yes, these arguments might help to avoid OOM errors.